web/gui/dashboard_info.js

```diff
@@ -504,6 +504,18 @@ netdataDashboard.menu = {
         title: 'VCSA',
         icon: '<i class="fas fa-server"></i>',
         info: 'vCenter Server Appliance health statistics. Data collected from <a href="https://vmware.github.io/vsphere-automation-sdk-rest/vsphere/index.html#SVC_com.vmware.appliance.health">Health API</a>.'
+    },
+
+    'zookeeper': {
+        title: 'Zookeeper',
+        icon: '<i class="fas fa-database"></i>',
+        info: 'Provides health statistics for <b><a href="https://zookeeper.apache.org/">Zookeeper</a></b> server. Data collected through the command port using <code><a href="https://zookeeper.apache.org/doc/r3.4.13/zookeeperAdmin.html#sc_zkCommands">mntr</a></code> command.'
+    },
+
+    'hdfs': {
+        title: 'HDFS',
+        icon: '<i class="fas fa-folder-open"></i>',
+        info: 'Provides <b><a href="https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html">Hadoop Distributed File System</a></b> performance statistics. Module collects metrics over <code>Java Management Extensions</code> through the web interface of an <code>HDFS</code> daemon.'
```

Comment:
I have a doubt related to this link, I saw that for zookeper you are using firstly the `index` ( https://zookeeper.apache.org/ ) and in front you use a link to the version 3.4.13(https://zookeeper.apache.org/doc/r3.4.13/zookeeperAdmin.html#sc_zkCommands ), on the other hand, when you are setting links to Hadoop, you set for an old version. Why are you not using here a new version like the 3.2 ( http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html )? I have the fear we are missing important information from the newest version.

Reply:
thx, changed links